gnovm/pkg/gnolang/machine.go

```diff
@@ -1970,15 +1970,44 @@ func (m *Machine) PushForPointer(lx Expr) {
 }
 
 func (m *Machine) PopAsPointer(lx Expr) PointerValue {
```

Comment:
Comment should say, pop a pointer for writing. All calls to this will end up writing to it.